src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
+
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatedStyle = {};
+  const removedEmptySpaces = sourceString
+    .split(';\n')
+    .filter((string) => string.includes(':'));
+
+  removedEmptySpaces.forEach((style) => {
+    const keyAndValue = style.split(':');
+    const key = keyAndValue[0].trim();
+    const clearedStyle = keyAndValue[1]
+      .trim()
+      .split(' ')
+      .map(prop => prop.trim());
```

Comment:
```suggestion
    const clearedStyle = keyAndValue[1].trim();
```

Reply:
It will not work 🤔 ?

Reply:
No, It will not work. Due to the reason that between style props there might be some white spaces which should be deleted

Reply:
Actually, directly for this task it works well, however, what would we do if we get style which has some white spaces between values
example: border: solid    2px black;